src/GZCTF/Controllers/ErrorController.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace GZCTF.Controllers;
+
+[ApiController]
+[Route("/error")]
+public class ErrorController : ControllerBase
+{
+    [HttpGet]
+    [HttpPost]
+    [HttpPut]
+    [HttpDelete]
```

Comment:
You can simply remove them if you don't need to specify the HTTP request method.